apps/schematic/api/schematic_api/controllers/schema_controller.py

```diff
@@ -48,6 +49,21 @@ def get_property_label(
     )
 
 
+def get_relationships(relationship_type, schema_url):  # noqa: E501
+    """Gets a subgraph containing all relationships of a given type
+
+    Gets a subgraph containing all relationships of a given type # noqa: E501
+
+    :param relationship_type: Type of relationship in a schema, such as requiresDependency
+    :type relationship_type: str
+    :param schema_url: The URL of a schema in jsonld form
+    :type schema_url: str
+
+    :rtype: Union[RelationshipsPage, Tuple[RelationshipsPage, int], Tuple[RelationshipsPage, int, Dict[str, str]]
+    """
+    return schema_controller_impl.get_relationships(relationship_type, schema_url)
```

Comment:
To me, get relationships implies something a bit different than what is happening here. 
Here you are retrieving the nodes that are connected by a given relationship type.

Reply:
The endpoint retrieves a pair of nodes connected by a relationship right? What would you call the two nodes then?

Reply:
lol, I left out a suggestion bc I couldn't think of anything elegant... but maybe something like, get_connected_nodes_by_relationship? are you limited by characters?

Reply:
to me a endpoint called get_relationships, I would expect to retrieve the types of relationships between node pairs,...which we can actually do, and why I got confused earlier. I thought you had created a second endpoint where you were supplying two nodes and were retrieving the relationship, but then I realized you were just specifying the returns.

Reply:
I think ConnectedNodes would be fine (RelationshipType would be a parameter)